packages/config/src/projects/layer2s/form.ts

```diff
@@ -0,0 +1,27 @@
+import { upcomingL2 } from './templates/upcoming'
+import { Layer2 } from './types'
+
+export const form: Layer2 = upcomingL2({
+  id: 'form',
+  display: {
+    name: 'Form',
+    slug: 'form',
+    description:
+      'Form is an Optimistic Rollup utilizing the OP Stack. The Form L2 is focused on bringing mass adoption and interoperability to the SocialFi category.',
+    purposes: ['Universal', 'Social'],
```

Comment:
Tagging @lucadonnoh as I've seen you approve on other PRs :)

Would it be possible to add `SocialFi` as a purpose? I'd be happy to do so in this PR Or open another if that would be preferrable.

Thanks!